dds_web/static/swagger.yaml

```diff
@@ -2,22 +2,40 @@
 # yamllint disable rule:braces
 openapi: 3.0.0
 info:
-  description: |
-    Documentation for the API of the Data Delivery System (DDS) in SciLifeLab
-  version: "2.6.0"
+  description: | # html syntax is allowed
+    Documentation for the API of the Data Delivery System (DDS) in SciLifeLab Data Centre. <br>
+    This documentation is mainly intended for internal use. But developers and researchers
+     can use it at their own risk.
+    However, DC recomends to always use the client and tools provided to units and researchers. <br>
+    <br><br>
+    <b>WARNING</b>: Some endpoints are not according to the OpenAPI standard, and therefore
+    cannot be documented properly. These endpoints are marked with <u>CHECK METHOD</u> and as deprecated.<br>
+    These endpoints will be changed in the future, and will be documented properly in the <b>url</b> hosted in
+    <a href="/api/documentation/v3"> api/documentation/v3</a> <br>
+    <br>
+    <b>USE</b>: Change the server in the list below depending on the instance you are using,
+    and click on the "Authorize" button to enter your user and password. <br>
+    The endpoint <b> /user/encrypted_token </b> is used to generate an encrypted token for the user. <br>
+    Add this returned token in the second section of the same "Authorize" button used before. <br>
+    Call the endpoint <b> /user/second_factor </b> to authenticate with MFA. <br>
+    Lastly, save again this second token in the same place as the first one obtained(login out the previous one). <br>
+    Now you can start using the API! <br>
```

Comment:
```suggestion
    <b>For interaction with the API resources</b>: <br> 
1. Change the server in the list below depending on the instance you are using,
    and click on the "Authorize" button to enter your user and password. <br>
    2. Use the endpoint <b> /user/encrypted_token </b> to generate an encrypted token for the user. <br>
    3. Add this returned token in the second section, <em> secondFactorToken  (http, Bearer) </em> of the same "Authorize" form used before. <br>
    4. Call the endpoint <b> /user/second_factor </b> to authenticate with MFA. This will return a new token.<br>
    5. Lastly, go back to the "Authorize" form, click "Logout" at the <em> secondFactorToken  (http, Bearer) </em> part, and use the second token from the previous step to login again. <br>

    Now you can start using the API! <br>
```